templates/partials/filterApplicantBar.phtml

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+    <div class="col-xs-8 center-block">
+        <button class="btn btn-primary pull-right" type="button" data-toggle="collapse" data-target="#collapseExample" aria-expanded="false" aria-controls="collapseExample">
+            Apply Filters
+        </button>
+        <div class="collapse" id="collapseExample">
+            <div class="well">
+                <label for="daterange">Application Date</label>
+                <input type="text" name="daterange" value="01/01/2018 - 01/15/2018" />
```

Comment:
can we have a placeholder in the data picker input?
input field needs to be wider to display full date range